ratatui-widgets/src/chart.rs

```diff
@@ -895,21 +928,45 @@ impl<'a> Chart<'a> {
             Alignment::Right => Alignment::Left,
         };
 
+        // The first label will be drawn in the space offered by y labels on the left
         Self::render_label(buf, labels.first().unwrap(), label_area, label_alignment);
 
+        // Finding positions for the labels by incrementing the
+        // `width_per_tick` each step is not a good solution because
+        // it accumulates rounding error at each iteration.
+        // A correct solution is to find the center position for the
+        // label box, i.e. where the tick mark should be, in terms
+        // of percentage of the graph area's width.
```

Comment:
Nit: for consistency with the rest of the lib, wrap comments at 100 chars. (I generally use https://marketplace.visualstudio.com/items?itemName=stkb.rewrap for this, Other editors / IDEs will have similar solutions.)

Why does this matter? Mostly it's an opinionated subjective preference. I don't like comments to take up large amounts of vertical space. 100 chars hits a good balance for most things in rust IMO while 80 chars often feels overly narrow.

Reply:
I committed the desired rewrapping.